pnpm-lock.yaml

```diff
@@ -0,0 +1,16622 @@
+lockfileVersion: '6.0'
```

Comment:
I suggest to ignore this file and not commit it to the repo. 

The reasoning I'm basing this suggestion on can be found here: 
https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514